xml/admin_gui_dashboard.xml

```diff
@@ -2324,9 +2324,13 @@ EOF
     </step>
     <step>
      <para>
-      Select a storage back-end&mdash;either <guimenu>CephFS</guimenu> or
-      <guimenu>Object Gateway</guimenu>.
+      Select a storage back-end.
      </para>
+      <important>
+        <para>
+          At this time, &ganesha; exports backed by the &ogw; are unsupported.
```

Comment:
This depends: we could also say "At this time, &ganesha; exports backed by &cephfs; are the only supported option." or something. @lproven what do we like?

@LenzGr @mgfritch

Reply:
How about "At this time, only NFS exports backed by CephFS are supported."

Reply:
I like @LenzGr's suggestion here. Anything else I can come up with is wordier.